<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WizardPageQuestion</class>
 <widget class="QWidget" name="WizardPageQuestion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>547</width>
    <height>308</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>27</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>10</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_question1">
       <property name="text">
        <string>Security Question 1</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="MyComboBoxQuestion" name="comboBox_question1">
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_answer1">
       <property name="text">
        <string>Security Answer 1</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="MyLineEdit" name="lineEdit_answer1">
       <property name="maxLength">
        <number>120</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_question2">
       <property name="text">
        <string>Security Question 2</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="MyComboBoxQuestion" name="comboBox_question2"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_answer2">
       <property name="text">
        <string>Security Answer 2</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="MyLineEdit" name="lineEdit_answer2">
       <property name="maxLength">
        <number>120</number>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_question3">
       <property name="text">
        <string>Security Question 3</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="MyComboBoxQuestion" name="comboBox_question3"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_answer3">
       <property name="text">
        <string>Security Answer 3</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="MyLineEdit" name="lineEdit_answer3">
       <property name="maxLength">
        <number>120</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>10</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>28</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyLineEdit</class>
   <extends>QLineEdit</extends>
   <header>mylineedit.h</header>
  </customwidget>
  <customwidget>
   <class>MyComboBoxQuestion</class>
   <extends>QComboBox</extends>
   <header>mycomboboxquestion.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
